src/build.py

```diff
@@ -255,6 +255,18 @@ def CopyLibraryToArchive(library, install_subdir=''):
   shutil.copy2(library, install_lib)
 
 
+def Git(cmd, **kwargs):
+  """Invoke git."""
+  return proc.check_output(
+      ['git'] + cmd, **kwargs, shell=sys.platform=='win32')
```

Comment:
Travis doesn't like this syntax. I think `**kwargs` has to go after all the manual keyword args?